clouddriver-ecs/src/main/java/com/netflix/spinnaker/clouddriver/ecs/deploy/validators/EcsCreateServerGroupDescriptionValidator.java

```diff
@@ -57,6 +57,16 @@ public void validate(List priorDescriptions, Object description, ValidationError
     validateCredentials(createServerGroupDescription, errors, "credentials");
     validateCapacity(errors, createServerGroupDescription.getCapacity());
 
+    if (createServerGroupDescription.getSubnetTypes() != null
+        && createServerGroupDescription.getSubnetTypes().size() > 0) {
+      if (StringUtils.isNotBlank(createServerGroupDescription.getSubnetType())) {
+        errors.rejectValue(
+            "subnetTypes",
+            errorKey + "." + "subnetTypes" + "." + "invalid",
+            "SubnetType cannot be specified when SubnetTypes is specified. Please use SubnetTypes");
```

Comment:
sorta-nit: I can see quick reading of this msg being confusing (the plural vs. singular is subtle). What do you think about:
"SubnetType _(string)_ cannot be specified when SubnetTypes _(list)_ is specified. Please use SubnetTypes"

Reply:
True! I'll update it :)

Reply:
Addressed